src/measure.js

```diff
@@ -310,4 +310,4 @@ function debugEdges(edges) {
 			painter.drawPolygon([edge.p1.x, edge.p1.y, edge.p2.x, edge.p2.y]);
 		}
 	}
-}
+}
```

Comment:
Please leave the final newline there (git doesn't really like files that don't end in a newline)